node/express/containerCosmosDBWithTests/Tests/functional_tests/sampleFunctionalTests.js

```diff
@@ -4,6 +4,10 @@ const {until} = require('selenium-webdriver');
 
 process.env['Path'] = process.env['Path'] + ';' + process.env['ChromeWebDriver'];
 
+async function sleep_for_milliseconds(milliseconds){
+	await sleep(milliseconds)
```

Comment:
is this method available in node/javascript

Reply:
Also why do you need the wrapper

Reply:
wrapper not needed, removed. added settimeout